packages/plugins/rubygem-foreman_resource_quota/rubygem-foreman_resource_quota.spec

```diff
@@ -18,13 +18,14 @@ BuildRequires: foreman-plugin >= %{foreman_min_version}
 Requires: ruby
 BuildRequires: ruby
 BuildRequires: rubygems-devel
+BuildRequires: (rubygem(foreman-tasks) >= 10.0 with rubygem(foreman-tasks) < 11)
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
 # end specfile generated dependencies
 
 # start package.json devDependencies BuildRequires
-BuildRequires: (npm(@babel/core) >= 7.7.0 with npm(@babel/core) < 8.0.0)
-BuildRequires: npm(@theforeman/builder) >= 12.0.1
+BuildRequires: (npm(@babel/core) >= 7.26.0 with npm(@babel/core) < 8.0.0)
+BuildRequires: npm(@theforeman/builder) >=12.0.1
```

Comment:
```suggestion
BuildRequires: npm(@theforeman/builder) >= 12.0.1
```
RPM is picky :/

Reply:
Yeah, thanks, I have already opened a PR in the plugin itself. I guess, we need to fix it there. I don't feel like manually adjusting it with every packaging PR :)

Reply:
https://github.com/ATIX-AG/foreman_resource_quota/pull/82

Reply:
Thanks, but we can fix it here and don't need a new release just for that :)

Reply:
OK, then let's fix it here and have it fixed for the new release